steps/measure.py

```diff
@@ -120,6 +120,7 @@ def evaluate_ansatz_based_cost_function(
     noise_model: Optional[str] = None,
     device_connectivity: Optional[str] = None,
     prior_expectation_values: Optional[str] = None,
+    **kwargs
```

Comment:
Since `kwargs` don't necessarily belong to `estimation_tasks_transformations`, I'd rather have a separate field `estimation_tasks_transformations_kwargs`, WDYT?

Reply:
Fixed in 8d523d33d3d7eae1bc7fe71b7e816aa1624b05e1, at least I hope the syntax is right...